crates/ruff_cli/src/printer.rs

```diff
@@ -163,6 +155,14 @@ impl Printer {
                                 fixables.unapplicable_unsafe
                             )?;
                         }
+                    } else {
+                        if fixables.applicable > 0 {
```

Comment:
nit: maybe combine the separate `else` and `if` into a single `else if`?

Reply:
Hm.. I think the intent here is clearer as written as the `else` is for `UnsafeFixes` modes.